app/views/govuk_publishing_components/components/_secondary_navigation.html.erb

```diff
@@ -0,0 +1,20 @@
+<%
+  add_gem_component_stylesheet("secondary-navigation")
+
+  id ||= nil
+  items ||= []
```

Comment:
Is it worth using the component wrapper here? I think the `shared_helper` would be needed on this component as well?

Reply:
Hi @AshGDS - not sure what the component wrapper is that you are referring to here. And also what the `shared_helper` does in this case.

Reply:
@davidtrussler The component wrapper is something built by Andy to standardise the way data attributes / classes / aria tags are added to the parent HTML tag of the component. The shared helper AFAIK allows margin bottom to be applied to the component for styling purposes. 

An example of both helpers exist in the password input component I added - see lines 9 to 16, as well as line 40: https://github.com/alphagov/govuk_publishing_components/blob/eacb1572137f5678ffe4dcbe376d75a3162bf8b4/app/views/govuk_publishing_components/components/_password_input.html.erb#L9

You would then need to reflect the existence of these helpers in your `.yml` file - you would add `uses_component_wrapper_helper: true` to the file, and give an example of your component with margin bottom:  https://github.com/alphagov/govuk_publishing_components/blob/eacb1572137f5678ffe4dcbe376d75a3162bf8b4/app/views/govuk_publishing_components/components/docs/password_input.yml#L41

A test of the margin functionality in your spec file would be needed as well.

Component wrapper docs: https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component-wrapper-helper.md

Shared helper docs: https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component_conventions.md#shared-helper

Though not sure these are necessary for all components - may be worth asking Andy.

Reply:
Thanks for that, Ash. Have now added the wrapper as suggested.